config/locales/en.yml

```
@@ -106,6 +106,13 @@ en:
   coronavirus_form:
     errors: {}
     questions:
+      are_you_off_work_ill:
+        title: "Are you off work because you’re ill or self-isolating?"
+        title_caption: "Going in to work"
+        options:
+          - "Yes"
+          - "No"
+        custom_select_error: "Select yes if you're off work because you're ill or self-isolating"
```

Tiny, pedantic point
```suggestion
        custom_select_error: "Select yes if you’re off work because you’re ill or self-isolating"
```